UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json

```diff
@@ -0,0 +1,1799 @@
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".Today"
+              }
+            }
+          ],
+          "text": "Date of encounter (MM/DD/YYYY):",
+          "type": "date",
+          "required": true
+        },
+        {
+          "linkId": "UROENC.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"UrologicalSuppliesPrepopulation\".AnswerYes"
+              }
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "code": "radio-button",
+                    "system": "http://hl7.org/fhir/questionnaire-item-control"
+                  }
+                ]
+              }
+            }
+          ],
+          "text": "Is this an evaluation of the patient’s need for urological supplies?",
+          "required": true,
+          "type": "choice",
+          "answerValueSet": "#yesno",
+          "item": [
+            {
+              "linkId": "UROENC.1a",
+              "text": "If No, purpose of the encounter",
+              "type": "text",
+              "require": true,
+              "enableWhen": [
+                {
+                  "question": "UROENC.1",
+                  "operator": "=",
+                  "answerCoding": {
+                    "code": "N"
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "DIAG",
+      "text": "Diagnoses",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "DIAG.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".CurrentDiagnoses"
+              }
+            }
+          ],
+          "text": "Current Patient Diagnoses",
+          "type": "open-choice",
+          "require": true,
+          "repeats": true,
+          "answerOption": []
+        },
+        {
+          "linkId": "UROQUAL.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".RelatedDiagnoese"
+              }
+            }
+          ],
+          "text": "Patient Qualifying Diagnoses",
+          "type": "open-choice",
+          "repeats": true,
+          "required": true,
+          "answerOption": []
+        }
+      ]
+    },
+    {
```

Comment:
Are we omitting this section? 
![Screen Shot 2020-05-06 at 9 42 50 AM](https://user-images.githubusercontent.com/58572522/81184141-0a51b680-8f7e-11ea-8b19-bbdee88c4047.png)